types/proto/narwhal.proto

```diff
@@ -5,90 +5,52 @@ syntax = "proto3";
 
 package narwhal;
 
-// TODO: Make this proto the source of truth for all relevant narwhal types.
-
-/* 
- * Proto wrapper for Narwhal type 
- *
- * pub struct CertificateDigest([u8; DIGEST_LEN])
- *
- */
 message CertificateDigest {
     bytes digest = 1;
 }
 
-/* 
- * Proto wrapper for Narwhal type 
- *
- * pub struct BatchDigest(pub [u8; crypto::DIGEST_LEN]);
- *
- */
 message BatchDigest {
     bytes digest = 1;
 }
 
-/* 
- * Proto wrapper for Narwhal type 
- *
- * pub struct Batch(pub Vec<Transaction>);
- *
- */
 message Batch {
     repeated Transaction transaction = 1;
 }
 
-/* 
- * Proto wrapper for Narwhal type 
- *
- * pub type Transaction = Vec<u8>;
- *
- */
 message Transaction {
     bytes transaction = 1;
 }
 
-/*
- * Proto wrapper for Narwhal type 
- *
- * pub enum BlockErrorType {
- *   BlockNotFound,
- *   BatchTimeout,
- *   BatchError,
- * }
- */
  enum CollectionErrorType {
     COLLECTION_NOT_FOUND = 0;
     COLLECTION_TIMEOUT = 1;
     COLLECTION_ERROR = 2;
 }
 
-/*
- * Proto wrapper for Narwhal type 
- *
- * pub struct BlockError {
- *     id: CertificateDigest,
- *     error: BlockErrorType,
- * }
- */
 message CollectionError {
     CertificateDigest id = 1;
     CollectionErrorType error = 2;
 }
 
-/*
- * Proto wrapper for Narwhal type
- *
- * pub struct BatchMessage {
- *   pub id: BatchDigest,
- *   pub transactions: Batch,
- * }
- *
- */
  message BatchMessage {
     BatchDigest id = 1;
     Batch transactions = 2;
 }
 
+message MultiAddr {
+    string address = 1;
+}
```

Comment:
If the `MultiAddr` in the end is just a `string` then I would recommend to just use directly the `string` type. Unless we want to do something with this struct - e.x write some `validation` method on top of the `MultiAddr` to confirm the validity of it - where I see use of it.

e.x
```
impl MultiAddr {
   fn validate(&self) {
       /// Do some validation etc...
   }
}
```

Reply:
Multiaddr also has a more compact binary format so this could also just be a bytes field